examples/github-appcards/README.md

```diff
@@ -206,6 +206,10 @@ MIRO_REDIRECT_URI=https://peaceful-fairy-c2e727.netlify.app/.netlify/functions/a
 <b>Make sure your URLs have https:// at the beginning, otherwise the OAuth flow will not work.</b>
 Also make sure that your MIRO_REDIRECT_URI ends in `.netlify/functions/authorize`.
 
+8. Lastly, you will need to change the URL in the GitHub action to point to your own deployed serverless functions. Point your GitHub action for `issues` to point to your `issues` endpoint from your serverless functions. The same should be done for `project-cards`.
+   Go to [.github/workflows/issues.yml](.github/workflows/issues.yml) and change the URL on [line 11](.github/workflows/issues.yml#L11).
+   Go to [.github/workflows/project-cards.yml](.github/workflows/project-cards.yml) and change the URL [on line 13](.github/workflows/project-cards.yml#L13).
+
```

Comment:
Looks a bit ugly. Are these "go to" supposed to be as list items?

![image](https://github.com/miroapp/app-examples/assets/1961590/beb24bb3-d664-4bad-91d7-3899d299fbff)

```suggestion
8. Lastly, you will need to change the URL in the GitHub action to point to your own deployed serverless functions. Point your GitHub action for `issues` to point to your `issues` endpoint from your serverless functions. The same should be done for `project-cards`.
    - Go to [.github/workflows/issues.yml](.github/workflows/issues.yml) and change the URL on [line 11](.github/workflows/issues.yml#L11).
    - Go to [.github/workflows/project-cards.yml](.github/workflows/project-cards.yml) and change the URL [on line 13](.github/workflows/project-cards.yml#L13).
```